views/includes/page-templates/trust-details.html

```diff
@@ -3,7 +3,27 @@
 
     {% include "includes/list/errors.html" %}
 
+    {# Check if there are any beneficial owners. If the length of the 'beneficialOwners' array is 0, then 'isTrustToBeCeased' will be true #}
+    {# and can be used to show the ceased date - only for 'review' mode in update journey #}
+    {% set isTrustToBeCeased = pageData.beneficialOwners | length == 0 %}
+
     {% if isReview %}
+      {% if isTrustToBeCeased and isFeatureFlagTrustsCeasedDateEnabled %}
+        <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+          <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+              Important
+            </h2>
+          </div>
+          <div class="govuk-notification-banner__content">
+            <p class="govuk-notification-banner__heading">
+              This trust is no longer associated with a beneficial owner and will be removed.</p>
+            <p>Review the trust information and make sure it is correct as at the date the trust stopped being associated with the overseas entity.</p>
+            <p>If you don't want to remove this trust, you need to change the nature of control for the appropriate <a class="govuk-notification-banner__link" href="">beneficial owner</a>.</p>
```

Comment:
Not your mistake but I'm not sure this text is 100% correct. Story says:

_Beneficial owner can be disassociated from the trust by either:
1. Ceased to being a beneficial owner to the overseas entity or
2. Change the natures of control from being a trustee of a trust._ 

So... it might not be the NOC that needs changing but rather the user might need to not cease the BO.

Reply:
I'll have to check this text with the analyst

Reply:
text updated (see Vernon's comment on ticket)